### PR TITLE
Fix tabpanex example

### PR DESCRIPTION
**Pull request by gwarf**
Fix tabpanex example.
Syntax highlighting wasn't working due to the suggested code formatting.

### CODE REVIEW

File: content/en/about/contributing/shortcodes/_index.md [hunk 1]
```diff
@@ -144,102 +142,107 @@ in multiple programming languages, you can use a tabbed pane for code snippets:
...
-}
-{{< /tab >}}
+{{< tabpane >}} {{< tab header="C++" lang="C++">}} #include <iostream> int
+main() { std::cout << "Hello World!" << std::endl; } {{< /tab >}}
+{{< tab header="Java" lang="Java">}} class HelloWorld { static public void main(
+String args[] ) { System.out.println( "Hello World!" ); } } {{< /tab >}}
+{{< tab header="Kotlin" lang="Kotlin">}} fun main(args : Array<String>) {
+println("Hello, world!") } {{< /tab >}} {{< tab header="Go" lang="Go">}} package
+main import "fmt" func main() { fmt.Printf("Hello World!\n") } {{< /tab >}}
 {{< /tabpane >}}
```
Comment (thebe14) [inline content/en/about/contributing/shortcodes/_index.md:152]: I strongly disagree on this. This is unreadable, unmanageable, and a nightmare to fix. You have no idea how long it took to get these examples to render "almost" correctly (working around the render problems of nested code blocks).

Comment (thebe14) [reply content/en/about/contributing/shortcodes/_index.md:152]: Not to mention that it renders wrong.

File: content/en/about/contributing/shortcodes/_index.md [hunk 2]
```diff
@@ -144,102 +142,107 @@ in multiple programming languages, you can use a tabbed pane for code snippets:
...
 ## Content with multiple variants
 
-When you need to include multiple variants of the same content, other than
-code snippets in multiple programing languages, you can use the follwing
-shortcode:
+When you need to include multiple variants of the same content, other than code
+snippets in multiple programing languages, you can use the follwing shortcode:
 
-```go-html-template
+````go-html-template
```
Comment (thebe14) [inline content/en/about/contributing/shortcodes/_index.md:159]: Why 4 backticks?

File: content/en/about/contributing/shortcodes/_index.md (hunk 1 above)
Comment (gwarf) [reply content/en/about/contributing/shortcodes/_index.md:172]: Sure, this should be fixed now: https://docs.egi.eu/documentation/416/about/contributing/shortcodes/#code-in-multiple-languages

File: content/en/about/contributing/shortcodes/_index.md [hunk 3]
```diff
@@ -144,102 +142,127 @@ in multiple programming languages, you can use a tabbed pane for code snippets:
...
+        }
+    }
+  {{< /tab >}}
+  {{< tab header="Kotlin" lang="Kotlin" >}}
+    fun main(args : Array<String>) {
+        println("Hello, world!")
+    }
+  {{< /tab >}}
+  {{< tab header="Go" lang="Go" >}}
+    import "fmt"
```
Comment (enolfc) [inline content/en/about/contributing/shortcodes/_index.md:167]: these spaces are rendered also, so it's not a good thing for python

File: content/en/about/contributing/shortcodes/_index.md [hunk 4]
```diff
@@ -30,8 +30,7 @@ The following code:
 
 Will render as:
 
-{{% pageinfo %}} This is a
-placeholder. {{% /pageinfo %}}
+{{% pageinfo %}} This is a placeholder. {{% /pageinfo %}}
```
Comment (enolfc) [inline content/en/about/contributing/shortcodes/_index.md:33]: This wrapping changes are a bit distracting. I understand they can happen but it's just adding noise

**Review by enolfc — COMMENTED** [review]
Just a comment on white spaces

File: content/en/about/contributing/shortcodes/_index.md (hunk 4 above)
Comment (gwarf) [reply content/en/about/contributing/shortcodes/_index.md:33]: It happened as the file wasn't formatted with prettier.

File: content/en/about/contributing/shortcodes/_index.md (hunk 3 above)
Comment (gwarf) [reply content/en/about/contributing/shortcodes/_index.md]: Good point, the indentation is causing troubles (as it was for the tabpanex), just removed it.

File: content/en/about/contributing/shortcodes/_index.md (hunk 2 above)
Comment (thebe14) [reply content/en/about/contributing/shortcodes/_index.md:179]: This is valuable information, maybe we should add it to the guide. E.g. I did not know this and had a hard time to make the examples render well.

**Review by enolfc — APPROVED** [review]
:+1:

**Pull request merged**